src/navigate-pop/viewing-sorting-options.md

```diff
@@ -0,0 +1,30 @@
+# Viewing & Sorting Options in Nautilus
+
+Users can change file icon size and sort files alphabetically, by modification date, file size, and type.
```

Comment:
> alphabetically, by modification date, file size, and type.

The "by" in the second list item is part of that list item and does not apply to the last two list items. One of the following would be more correct (I'm also going with "or" since only one sorting option works at a time in Nautilus):

> alphabetically, by modification date, by file size, or by type.

> alphabetically, or by modification date, file size, or type.